tutorial/BUILD

```diff
@@ -23,6 +23,7 @@ cc_binary(
     name = "101_quantity_makers",
```

Comment:
Q: Any reason this is a cc_binary and not a `cc_library` + `cc_test`?

Reply:
I had to dig a bit to jog my memory.  I think I did it this way because of the `print_raw_number_and_quantity()` function, which simply gets called and isn't part of a test.  Not a super compelling reason, but we can revisit it later.